src/postbacks.js

```diff
@@ -0,0 +1,27 @@
+import connection from './connection/connection'
+import {
+  SOCKET_POSTBACK
+} from './constants'
+
+class Postbacks {
+
+  send(postbackPayload) {
+    return new Promise((resolve, reject) => {
+      if(!postbackPayload || typeof postbackPayload !== 'string') {
```

Comment:
Shouldn't we receive validation only from server OR by the use of AJV schemas? I'd delete it here since we would have to keep amio-chat and these lib validations consistent.

Reply:
Good point, will remove the type validation